subcommands/version/tuf_repo.go

```diff
@@ -0,0 +1,10 @@
+package version
+
+import _ "embed"
+
+// This is generated with:
+//  git show tuf-metadata:repository/root.json | json_pp > ./subcommands/version/root.json
+//go:embed root.json
+var RootContent []byte
+
+const RootUrl = "https://raw.githubusercontent.com/foundriesio/fioctl/tuf-metadata/repository/"
```

Comment:
IIUC this is the TUF repo URL, not just the root URL.
So, I'd name it like `TufRepoUrl`.